tests/ui/test_validation_ui.py

```diff
@@ -47,6 +55,20 @@ def test_dashboard_validation_ui(self, setup_ui_class):
         else:
             validation_ui_obj.verification_ui()
 
```

Comment:
Please add a comment mentioning below that the below verification was added to cover BZ 2155743

Reply:
done